src/neynar-api/v1/client.ts

```diff
@@ -73,7 +73,8 @@ export class NeynarV1APIClient {
     if (axiosInstance === undefined) {
       axiosInstance = axios.create({
         headers: {
-          'x-sdk-version': version
+          'x-sdk-version': version,
+          'x-api-key': apiKey,
```

Comment:
@eurbs You can remove this line, it'll work regardless of this, cuz we are assigning apiKey when we instantiate a config object

```
new Configuration({
      basePath: basePath ? `${basePath}/v1` : BASE_PATH,
      apiKey: apiKey,
    })
```